fs-v1.go

```diff
@@ -48,6 +53,7 @@ func newFSObjects(disk string) (ObjectLayer, error) {
 
 	// Runs house keeping code, like creating minioMetaBucket, cleaning up tmp files etc.
 	fsHouseKeeping(storage)
+	createMetaFormatJSON(storage)
```

Comment:
Handle the errors and return. do not ignore it. It is okay for 'fsHouseKeeping()'